### PR TITLE
Kb bookbyen

### PR DESCRIPTION
**Pull request by kasperbirch1**
Adds a new slide.

This is a new slide for Book-byen. The Slide act as a calendar to display information about what is going on in which room.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ const entry = devMode
       examples: path.resolve(__dirname, "./src/index.js"),
     }
   : {
+     "book-byen": path.resolve(__dirname, "./src/book-byen/book-byen.js"),
      twentyThreeVideo: path.resolve(__dirname, "./src/twenty-three-video/twenty-three-video.js"),
```
Comment (kasperg) [inline webpack.config.js]: Please keep entries consistent. It seems like kebab-case is the default. In that case it should be something like `twenty-three-video`.

File: src/twenty-three-video/twenty-three-video.js [hunk 2]
```diff
@@ -41,28 +40,25 @@ function TwentyThreeVideo({ slide, content = {}, run, slideDone }) {
     .map((item) => item.trim())
     .filter(Boolean);
 
-  const [currentVideoIndex, setCurrentVideoIndex] = useState(0)
+  const [currentVideoIndex, setCurrentVideoIndex] = useState(0);
   const [videoId, setVideoId] = useState(formattedVideoList[currentVideoIndex]);
 
+  /** @param e */
   function videoEndedEvent(e) {
```
Comment (kasperg) [inline src/twenty-three-video/twenty-three-video.js:47]: Why `/** @param e */`?

File: build/book-byen-admin.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
+[
```
Comment (kasperg) [inline build/book-byen-admin.json:1]: Consider [marking everything in the `build` dir as generated](https://github.com/github/linguist/blob/master/docs/overrides.md).

File: src/book-byen/book-byen.scss [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    font-size: 0.6em;
+  }
+}
+
+.bookbyen__subslide {
+  width: 100%;
+  display: grid;
+  align-items: center;
+  justify-items: right;
+}
```
Comment (kasperg) [inline src/book-byen/book-byen.scss]: Please move these classes under `.bookbyen` like we do elsewhere in this file with `&__subslide`.

File: src/book-byen/book-byen.scss [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+  &__logo {
+    width: 3em;
+    justify-self: right;
+  }
+}
+
+.bookbyen-top__place--with-filters {
+  font-size: 1.2em;
+
+  .bookbyen-top__place-name {
```
Comment (kasperg) [inline src/book-byen/book-byen.scss:162]: See comment below.

File: src/book-byen/book-byen-admin.json [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    "text": "Indhold",
+    "name": "header2",
+    "formGroupClasses": "h5 mb-3"
+  },
+  {
+    "key": "book-byen-form-3",
+    "input": "textarea",
+    "name": "title",
+    "label": "Overskrift på slide",
+    "helpText": "Her kan du skrive overskriften til slidet",
```
Comment (kasperg) [inline src/book-byen/book-byen-admin.json:21]: There are a few entries here: `text`, `name`, `label` and `helpText`. The usage, language and "quality" of the content of each of these entries all through the file vary quite a bit.

Please see if you can streamline these.

File: src/book-byen/book-byen.js [hunk 7]
```diff
@@ -0,0 +1,260 @@
...
+};
+
+const filterEvents = (item) => {
+  if (item?.isDeleted) {
+    // Filter the item away.
+    return false;
+  }
+  // Keep.
+  return true;
+};
```
Comment (kasperg) [inline src/book-byen/book-byen.js]: Can we simplify this?

```suggestion
const filterEvents = (item) => {
  return item?.isDeleted;
};
```

**Review by kasperg — APPROVED** [review]
:+1: I have some comments for you to look into before you merge.

File: src/twenty-three-video/twenty-three-video.js (hunk 2 above)
Comment (googletorp) [reply src/twenty-three-video/twenty-three-video.js:47]: I might have done a lint:fix :)

File: src/book-byen/book-byen-admin.json (hunk 6 above)
Comment (googletorp) [reply src/book-byen/book-byen-admin.json:21]: I'll handle this in a follow up when I know what should actually happen

**Pull request merged**